### ISSUE TITLE
Source code for Goobi/WEB-INF/lib/goobi-api.jar is missing

### ISSUE THREAD

**Issue by stweil** [kitodo/kitodo-production#188]
The source code for this Java binary (or at least parts of it) is missing. Obviously this issue was already addressed in 2012, but that patch never made it into the current version, see https://lists.launchpad.net/slub.team/msg00214.html.

**Comment by stweil** [comment]
Is there a chance that this issue can be resolved? As far as I know this is one of two remaining libraries (jar files) with unclear source code.

**Comment by stweil** [comment]
It looks like this file no longer exists. It was renamed to Kitodo/src/main/webapp/WEB-INF/lib/goobi-api.jar, and commit 54073d082264865ad15d7ee160162f8e55b302a8 says that it was moved to the Slub-Nexus Maven Repository. I am not sure what happened then.

**Comment by henning-gerhardt** [comment]
This jar file is / was only needed in 2.x and was removed in development of 3.x as obsolete and not in use.

Maybe closing all old 2.x issues as development of 2.x has end and no future release will come @solth ?

**Comment by solth** [comment]
Yes, that's true. I have already started closing issues that definitely only related to 2.x (see https://github.com/kitodo/kitodo-production/issues?q=is%3Aissue+is%3Aclosed+label%3A2.x). 

I did not close issues where I wasn't sure if they still had any relevance for Kitodo.Production 3.x, though. Reading your comments for this issue in particular it seems this is _not_ the case here and we can indeed close it.
